.github/workflows/os_java_ci.yml

```
@@ -20,19 +20,20 @@ defaults:
 
 jobs:
   os-java:
-    name: OS x Java
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest]
-        java-version: [8, 11]
-        maven-version: ['3.8.1']
+      os: [ubuntu-latest, windows-latest]
+      java-version: [8, 11]
+      maven-version: ['3.8.1']
```

The workflow run complains that the file is not valid.
```suggestion
        os: [ubuntu-latest, windows-latest]
        java-version: [8, 11]
        maven-version: ['3.8.1']
```